### PR TITLE
fix typo of urlRewrite

### PR DESCRIPTION
**Pull request by woodgear**
**What type of PR is this?**
/kind documentation

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

File: apis/v1alpha2/httproute_types.go [hunk 1]
```diff
@@ -523,8 +523,7 @@ type HTTPRouteFilter struct {
 	// +optional
 	RequestRedirect *HTTPRequestRedirectFilter `json:"requestRedirect,omitempty"`
 
-	// URLRewrite defines a schema for a filter that responds to the
-	// request with an HTTP redirection.
+	// URLRewrite defines a schema for a filter that rewrite http request's url.
```
Comment (hbagdi) [inline apis/v1alpha2/httproute_types.go:526]:
```suggestion
	// URLRewrite defines a schema for a filter that rewrites an HTTP request's URL.
```

Comment (woodgear) [reply apis/v1alpha2/httproute_types.go]: update

File: apis/v1alpha2/httproute_types.go [hunk 2]
```diff
@@ -523,8 +523,7 @@ type HTTPRouteFilter struct {
 	// +optional
 	RequestRedirect *HTTPRequestRedirectFilter `json:"requestRedirect,omitempty"`
 
-	// URLRewrite defines a schema for a filter that responds to the
-	// request with an HTTP redirection.
+	// URLRewrite defines a schema for a filter that rewrites an HTTP request's URL.
```
Comment (bowei) [inline apis/v1alpha2/httproute_types.go:526]: Let's use the same language as the full description comment:

`RequestRedirect defines a filter that modifies a request during forwarding.`

**Comment by robscott** [comment]
Hey @woodgear, thanks for fixing this! Do you have time to address the last comment on this PR?

File: apis/v1alpha2/httproute_types.go (hunk 2 above)
Comment (woodgear) [reply apis/v1alpha2/httproute_types.go:526]: i don't get it. should i change the comment in URLRewrite to "URLRewrite defines a filter that modifies a request during forwarding." ? when mean "the same language as the full description comment"?

**Comment by woodgear** [comment]
> Hey @woodgear, thanks for fixing this! Do you have time to address the last comment on this PR?

update.

**Pull request merged**